coderd/notifications/notifier.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/coder/coder/v2/coderd/database"
 )
 
+const (
+	notificationsDefaultLogoURL = "https://coder.com/coder-logo-horizontal.png"
```

Comment:
Can we instead link to something hosted on the Coder deployment itself?

Reply:
I believe @defelmnq created a follow-up for this already?

Reply:
Ah thanks, I must have missed it in the above discussion.